consolelabs/mochi-api
# perf: add gorm resolver to read replica db

**What does this PR do?**

-   [x] add gorm resolver to read replica db

**How to test**

-   run docker-compose.replica.yml to create master, slave db instances.
- call any GET API to see changes
